docs/documentation/algorithms.md

```diff
@@ -5,7 +5,7 @@
 
 ## Algorithms ensembles
 
-The HES-XPLAIN platform offers various tools to extract explainability and much more features: 
+Here you can find an overview of the different algorithm ensembles. They are contained by a repository called `Dimlpfidex` and are referred to as a whole by this term. Every one of them can be accessed through the links below. 
```

Comment:
* Do you mean "in a repository"?
* Also, I'd remove the superfluous wording. Try something like: "They are contained in the Dimlpfidex repository and are collectively referred to by this term".
* add a link to the repo directly on "dimlpfidex".